### ISSUE TITLE
Feature: Add copy animation to Git clone flyout

### ISSUE THREAD

**Issue by Lamparter** [0x5bfa/FluentHub#506]
### 📄 Description

Currently the activation of the git command copy is not obvious currently. This should be made more obvious for accessibility

#### 📋 High Priority
- [ ] Choose between static/animated copy icons
- [ ] Research copy animations/icons
- [ ] Implement
